docs: add note in CI overview about how to disable dbus (#6274)
cypress-io/cypress-documentation: docs/app/continuous-integration/overview.mdx

NO_COLOR=1 cypress run
```

### `Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket`

This error message indicates that Electron is attempting to connect to `dbus`, a desktop environment service, when there is no `dbus` present.

> Cypress does its best to detect non-interactive shells and disable `dbus` by setting the `DBUS_SESSION_BUS_ADDRESS` environment variable to `disabled:`.

Reviewer: ```suggestion
Cypress attempts to detect non-interactive shells and disable `dbus` by setting the `DBUS_SESSION_BUS_ADDRESS` environment variable to `disabled:`.
```
